package.json

```diff
@@ -317,9 +317,9 @@
           "scope": "window"
         },
         "xml.completion.autoCloseRemovesContent": {
-          "type": "boolean",
+          "type": "boolean",  
```

Comment:
Remove the trailing whitespace here. Take a look into setting up your editor to trim trailing whitespace when you save. This helps to avoid committing trailing whitespace.